even_fib_nums.js

```diff
@@ -6,24 +6,52 @@
  * @return {Number} sum
  */
 function _sumFibs( maxFibValue ) {
+
   let sum = 0;
+  let arr = [1, 2];
```

Comment:
Try doing this without using an array but rather three variables, one to keep track of the current fib number, and two others to keep track of the previous two fib numbers.